src/game/sa1_sa2_shared/item_box.c

```diff
@@ -72,7 +72,11 @@ const u16 ItemBox_randTypeTable[8] = { ITEM__RINGS_RANDOM,  ITEM__MP_RANDOM, ITE
                                        ITEM__INVINCIBILITY, ITEM__MP_RANDOM, ITEM__RINGS_5,  ITEM__MP_RANDOM };
 
 const u16 ItemBox_subRandTypeTable[] = { ITEM__MP_SWAP, ITEM__MP_CONFUSION, ITEM__MP_SLOWDOWN, ITEM__MP_ATTACK };
-const u8 ItemBox_ringAmountTable[] = { 1, 5, 10, 30, 50, 0, 0, 0 };
+#if (GAME == GAME_SA1)
+const u8 ItemBox_ringAmountTable[] = { 1, 5, 10, 20, 30, 50 };
```

Comment:
Could make the same definition with just an if for the single value in the middle?
```suggestion
const u8 ItemBox_ringAmountTable[] = { 1, 5, 10, 20, 
#if (GAME == GAME_SA1)
30,
#endif
50 };
```

Reply:
The 20 is not in SA2, as you might be able to see. Generally I find it more readable to have to whole array in one line.

And yeah, it's safer to use ARRAY_COUNT(ItemBox_ringAmountTable), instead of magic numbers, so I opted for removing the zeroes.
Hence the ALIGNED(4) in `music_manager.c`.
Doesn't match otherwise, curious as it may be.